server/controller/src/main/java/ai/starwhale/mlops/domain/model/mapper/ModelVersionMapper.java

```diff
@@ -69,8 +69,8 @@ List<ModelVersionEntity> list(@Param("modelId") Long modelId,
     @Options(useGeneratedKeys = true, keyColumn = "id", keyProperty = "id")
     int insert(ModelVersionEntity version);
 
-    @Update("update model_version set status = #{status} where id = #{id}")
-    int updateStatus(@Param("id") Long id, @Param("status") Integer status);
```

Comment:
Could we gurantee that `versionName` will always be unique in the future especially crossing all projects. For example, after copy model cross projects.

Reply:
fixed